GNUmakefile

```diff
@@ -114,7 +114,7 @@ generate: abigen codecgen mockery ## Execute all go:generate commands.
 testscripts: chainlink-test ## Install and run testscript against testdata/scripts/* files.
 	go install github.com/rogpeppe/go-internal/cmd/testscript@latest
 	find testdata/scripts -type d | xargs -I % \
-	sh -c 'ls %/*.txtar > /dev/null 2>&1 || return 0 && PATH=$(CURDIR):$(PATH) testscript -e COMMIT_SHA=$(COMMIT_SHA) -e VERSION=$(VERSION) $(TS_FLAGS) %/*.txtar'
+	sh -c 'ls %/*.txtar > /dev/null 2>&1 || return 0 && PATH=$(CURDIR):$(PATH) testscript -e HOME="$(TMPDIR)/home" -e COMMIT_SHA=$(COMMIT_SHA) -e VERSION=$(VERSION) $(TS_FLAGS) %/*.txtar'
```

Comment:
Note: we don't have access to the $WORK var from outside testscript, so I used $TMPDIR instead.

This is... likely to be dodge since we won't get the test isolation we normally get when running these testscripts ($WORK is different per test normally).